<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dodajdobazy</class>
 <widget class="QDialog" name="dodajdobazy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbSecDodaj">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAnuluj">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teMarka">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teModel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teNrRej">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Marka:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>NrRej:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
